logging/images/logstash/logstash.conf

```diff
@@ -0,0 +1,13 @@
+input {
+  syslog { port => 51415 }
+}
+
+output {
+  elasticsearch {
+    hosts => ["elasticsearch:9200"]
+  }
+  # Remove in production
```

Comment:
I would suggest removing (or commenting) this one. It's good for demo purpose but in production it would be very heavy on resources to output everything to stdout.